backend/api/Database/Models/Asset.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+#pragma warning disable CS8618
+namespace Api.Database.Models
+{
+    public class Asset
+    {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        public string Id { get; set; }
+
+        [Required]
+        [MaxLength(200)]
+        public string Name { get; set; }
+
+        [Required]
+        [MaxLength(10)]
+        public string ShortName { get; set; }
```

Comment:
```suggestion
        public string AssetCode { get; set; }
```
So that we keep consistency with other places in the code and with Equinor naming system.